src/launchpad/sentry_sdk_init.py

```diff
@@ -0,0 +1,90 @@
+"""Sentry SDK initialization for the Launchpad service."""
+
+from __future__ import annotations
+
+import logging
+import os
+
+from typing import Any, Dict
+
+import sentry_sdk
+
+from sentry_sdk.integrations.aiohttp import AioHttpIntegration
+from sentry_sdk.integrations.asyncio import AsyncioIntegration
+from sentry_sdk.integrations.logging import LoggingIntegration
+from sentry_sdk.integrations.stdlib import StdlibIntegration
+from sentry_sdk.integrations.threading import ThreadingIntegration
+from sentry_sdk.types import Event
+
+from launchpad.utils.logging import get_logger
+
+logger = get_logger(__name__)
+
+
+class LaunchpadInternalError(Exception):
+    """Base exception for launchpad internal errors that should be excluded from Sentry reporting."""
+
+    pass
+
+
+def initialize_sentry_sdk() -> None:
+    """Initialize Sentry SDK with launchpad-specific configuration."""
+    config = get_sentry_config()
+
+    # Only initialize Sentry SDK in production environment
+    if config.get("environment") != "production":
+        logger.debug(f"Not in production environment ({config.get('environment')}), skipping Sentry SDK initialization")
+        return
+
+    # Skip initialization if DSN is not provided
+    if not config.get("dsn"):
+        logger.info("Sentry DSN not provided, skipping Sentry SDK initialization")
+        return
+
+    def before_send(event: Event, hint: dict[str, Any]) -> Event | None:
+        """Filter out internal errors from Sentry reporting."""
+        if "exc_info" in hint:
+            exc_type, exc_value, tb = hint["exc_info"]
+            # Exclude errors intended for internal handling, not Sentry
+            if isinstance(exc_value, LaunchpadInternalError):
```

Comment:
What's the purpose of this, are there types of errors we don't want to report? I don't see LaunchpadInternalError used anywhere yet. Would be nice to have an example of what you intend to do with this b/c I can't think of any right now.

Reply:
yeah frankly i can't either. I added it pre-emptively. I'll just remove it for now